documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -1,65 +1,70 @@
 Execution Policies
 ##################
 
-The implementation supports the device execution policies used to run the massive parallel
-computational model for heterogeneous systems. The policies are specified in
-the |onedpl_long| (|onedpl_short|) section of the `oneAPI Specification
-<https://spec.oneapi.io/versions/latest/elements/oneDPL/source/parallel_api.html#dpc-execution-policy>`_.
-
-For any of the implemented algorithms, pass one of the execution policy objects as the first
-argument in a call to specify the desired execution behavior. The policies have
-the following meaning:
-
-================================= ==============================
-Execution Policy Value            Description
-================================= ==============================
-``seq``                           Sequential execution.
---------------------------------- ------------------------------
-``unseq``                         Unsequenced SIMD execution. This policy requires that
-                                  all functions provided are SIMD-safe.
---------------------------------- ------------------------------
-``par``                           Parallel execution by multiple threads.
---------------------------------- ------------------------------
-``par_unseq``                     Combined effect of ``unseq`` and ``par``.
---------------------------------- ------------------------------
-``dpcpp_default``                 Massive parallel execution on devices using |dpcpp_short|.
---------------------------------- ------------------------------
-``dpcpp_fpga``                    Massive parallel execution on FPGA devices.
-================================= ==============================
+According to `the oneAPI specification
+<https://uxlfoundation.github.io/oneAPI-spec/spec/elements/oneDPL/source/index.html>`_,
+|onedpl_long| (|onedpl_short|) provides execution policies semantically aligned with the C++ standard,
+also referred to as *standard aligned* or *host execution policies*, and the *device execution policies*
+to run data parallel computations on heterogeneous systems.
```

Comment:
It may be more clear if we make this into two sentences.  Change the comma to a period then start the next sentence as "|onedpl_short| also provides the *device execution policies* ..."

Reply:
OK, I will think of it - but perhaps in another patch, unless bigger changes are requested for this one.

Reply:
not required, the rest LGTM.  About to approve.

Reply:
Changed "and ..." to "as well as device execution policies", but otherwise left it as is. I do not want to split, as the "According to the spec" part applies to both policy kinds.